conf/defaults.yaml

```diff
@@ -231,16 +228,13 @@ topology.multilang.serializer: "org.apache.storm.multilang.JsonSerializer"
 topology.shellbolt.max.pending: 100
 topology.skip.missing.kryo.registrations: false
 topology.max.task.parallelism: null
-topology.max.spout.pending: null
+topology.max.spout.pending: null  # TODO: We dont need this any more
 topology.state.synchronization.timeout.secs: 60
-topology.stats.sample.rate: 0.05
+topology.stats.sample.rate: 0.001
```

Comment:
Ok I personally feel that sampling can be reduced a bit, but if you have a preference towards retaining it .... i am ok with that.  JCQueue is at the core messaging system .. not separate. the numbers demonstrate the improvement.